README.md

```diff
@@ -1,2 +1,462 @@
-# certificate
+# Certificate System Role
+
 Role for managing TLS/SSL certificate issuance and renewal
+
+Linux system role to issue and renew SSL certificates.
+
+Basic usage:
+
+```yaml
+
```

Comment:
remove this empty line

Reply:
I removed all leading and trailing empty lines from the examples